ui/src/audioplayer/PlayerToolbar.js

```diff
@@ -23,10 +23,9 @@ const Toolbar = ({ id }) => {
   const handlers = {
     TOGGLE_LOVE: useCallback(() => toggleLove(), [toggleLove]),
   }
-
   return (
     <ThemeProvider theme={createMuiTheme(theme)}>
-      <QualityInfo record={data} />
+      {data && <QualityInfo source="quality" record={data} sortable={false} />}
```

Comment:
In the `PlayerToolbar` I don't think `source` is required. Please try removing and if it does not cause any issues.